lib/ClimaLandSimulations/src/Fluxnet/Fluxnet.jl

```diff
@@ -0,0 +1,64 @@
+module Fluxnet
+
+using ClimaLandSimulations
+import SciMLBase
+import ClimaTimeSteppers as CTS
+using ClimaCore
+import CLIMAParameters as CP
+using Statistics
+using Dates
+using Insolation
+using StatsBase
+using Interpolations
+using ClimaLand
+using ClimaLand.Domains: Column
+using ClimaLand.Soil
+using ClimaLand.Soil.Biogeochemistry
+using ClimaLand.Canopy
+using ClimaLand.Canopy.PlantHydraulics
+import ClimaLand.Parameters as LP
+import ClimaComms
+context = ClimaComms.context()
+using Unitful: R, L, mol, K, kJ, °C, m, g, cm, hr, mg, s, μmol, Pa, W, mm, kPa
+using UnitfulMoles: molC
+using Unitful, UnitfulMoles
+using ArtifactWrappers
+using DelimitedFiles
+using Thermodynamics
+using Formatting
+using DataFrames
+
+# UnitfulMoles compound
+@compound CO₂

```

Comment:
could this be moved to a file within the module? maybe fluxnet_utilities/site_parameters.jl?

Reply:
These are small changes but if you dont mind, changing them in this PR might be better since some of these lines may not show up in later PRs, and it will be hard to keep a list of all of them on your own.

Reply:
I put this line in inputs_dataframe.jl, where it is needed